reemgawad/brunchtime
# Add seated alert

What changed:

- Added an alert (bootstrap modal) for "Are you seated?", on the show page of visit (directions page)
- Added destroy route/action for a visit when user clicks "I'm not going" on the alert

How to test:
GOTO a restaurant -> click "Take me there" -> click button "Launch Modal" -> click "I'm not going". Should redirect back to restaurant index.
To check if visit was deleted, try going back to that visit ID in the browser
